lib/notification_service/notification_service/cli/cli_parser.py

```diff
@@ -121,11 +122,113 @@ class GroupCommand(NamedTuple):
 
 CLICommand = Union[ActionCommand, GroupCommand]
 
-VERSION_COMMAND = ActionCommand("version",
-                                "Shows the version of Notification.",
-                                lazy_load_command("notification_service.cli.commands.version_command.version"),
-                                [],
-                                "Shows the version of Notification.")
+ARG_SERVER_URI = Arg(
+    ("-s", "--server-uri"),
```

Comment:
The `server-uri` is positional argument, not the named argument.

Reply:
The `server-uri` is optional in the future. We will have notification client configuration, and the `server-uri` is only used to override configuration.

Reply:
@jiangxin369 , at present, the `server-uri` is not optional. Therefore, this could be positional argument, and in feature make this optional with the client configuration.